File: fix_votes.js
const fs = require('fs');

const env = {};
const lines = fs.readFileSync('.env.local', 'utf-8').split('\n');
for (const line of lines) {
    if (line.includes('=')) {
        const parts = line.split('=');
        env[parts[0].trim()] = parts.slice(1).join('=').trim();
    }
}

const url = `${env['NEXT_PUBLIC_SUPABASE_URL']}/rest/v1/votes_cast?status=eq.pending`;

fetch(url, {
    method: 'PATCH',
    headers: {
        'apikey': env['SUPABASE_SERVICE_ROLE_KEY'],
        'Authorization': `Bearer ${env['SUPABASE_SERVICE_ROLE_KEY']}`,
        'Content-Type': 'application/json',
        'Prefer': 'return=representation'
    },
    body: JSON.stringify({ status: 'valid' })
})
.then(res => res.json())
.then(data => console.log('Updated:', data))
.catch(err => console.error(err));

File: fix_votes.py
import os
import requests
import json

env = {}
with open('.env.local', 'r') as f:
    for line in f:
        if '=' in line:
            key, val = line.strip().split('=', 1)
            env[key] = val

url = f"{env['NEXT_PUBLIC_SUPABASE_URL']}/rest/v1/votes_cast?status=eq.pending"
headers = {
    "apikey": env['SUPABASE_SERVICE_ROLE_KEY'],
    "Authorization": f"Bearer {env['SUPABASE_SERVICE_ROLE_KEY']}",
    "Content-Type": "application/json",
    "Prefer": "return=representation"
}

res = requests.patch(url, headers=headers, json={"status": "valid"})
print("Status:", res.status_code)
print("Response:", res.text)
